powershell-adapter/Tests/powershellgroup.resource.tests.ps1

```diff
@@ -5,7 +5,7 @@ Describe 'PowerShell adapter resource tests' {
 
     BeforeAll {
         $OldPSModulePath  = $env:PSModulePath
-        $env:PSModulePath += [System.IO.Path]::PathSeparator + $PSScriptRoot
+        $env:PSModulePath += [System.IO.Path]::PathSeparator + $PSScriptRoot + [System.IO.Path]::DirectorySeparatorChar
```

Comment:
```suggestion
        $env:PSModulePath += [System.IO.Path]::PathSeparator + $PSScriptRoot
```

Was there a need to add this?

Reply:
was part of troubleshooting. not needed.